packages/framework/src/Resources/config/services_test.yml

```diff
@@ -210,7 +210,8 @@ services:
     Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade: ~
 
     # Prevent listening to kernel.response event while running tests
-    Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportListener: ~
+    Shopsys\FrameworkBundle\Model\Product\Search\Export\ProductSearchExportListener:
+        tags: []
```

Comment:
Was this necessary? Why?

Reply:
It’s not necessary, just wanted to be clear about intentions. That it’s necessary to remove tag.